### ISSUE TITLE
implement sats

### ISSUE THREAD

**Issue by nplanel** [emoon/capstone#34]
```
-   31054:      4c82            satsl %d2
+   31054:      4c82            .short 0x4c82
```
seems a Coldfire v4 instruction 'Signed Saturate'
others was introduce in the v4 : MOVE3Q, MOVSX, MOVZX

**Comment by emoon** [comment]
I see. I have never used that one (but I have only coded 68k for Amiga and not Coldfire)

**Comment by nplanel** [comment]
me too :) (Atari ST) but I compare with objdump, so obviously we miss some new intructions

**Comment by emoon** [comment]
I wonder if we should at new field here https://github.com/emoon/capstone/blob/next/include/capstone/capstone.h#L106 when having support for Coldfire instructions? dunno

**Comment by nplanel** [comment]
Yes that make sense, but how many version/flavor of Coldfire exist ?
We need to look more binutils source code ...

**Comment by emoon** [comment]
I have no idea. I would be fine with not supporting Coldfire specific instruction for the first PR also.
